### PR TITLE
DevWs for frient containing containing SmartSense Open/Closed Sensor

### PR DESCRIPTION
**Pull request by frient-design**

### CODE REVIEW

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 1]
```diff
@@ -41,8 +41,7 @@ metadata {
...
         fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "ADUROLIGHT", model: "CSW_ADUROLIGHT", deviceJoinName: "ERIA Open/Closed Sensor", mnmn: "SmartThings", vid: "generic-contact-3" //ERIA Contact Sensor V2.0
 		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "Sercomm Corp.", model: "SZ-DWS04", deviceJoinName: "Sercomm Open/Closed Sensor", mnmn: "SmartThings", vid: "generic-contact" //Sercomm Door Window Sensor
-        //Dawon
-        fingerprint inClusters: "0000, 0003, 0006, 0500", outClusters: "0003, 0019", manufacturer: "DAWON_DNS", model: "SS-B100-ZB", deviceJoinName: "Dawon Signal Interlock", mnmn: "0AIg", vid: "dawon-zigbee-signal-interlock2"
+		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0001,0003,000F,0020,0500", outClusters: "000A,0019", manufacturer: "frient A/S", model :"WISZB-120", deviceJoinName: "frient Entry Sensor Pro"
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: Don't delete another device's fingerprint, add your own below.

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 2]
```diff
@@ -210,13 +209,15 @@ def configure() {
 	def cmds = refresh() +
 		zigbee.configureReporting(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS, DataType.BITMAP16, 30, 60 * 5, null) +
 		zigbee.batteryConfig() +
-		zigbee.temperatureConfig(30, 60 * 30) +
+        zigbee.temperatureConfig(30, 60 * 30) +
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: This transition from tabs to spaces here is unnecessary

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 3]
```diff
@@ -210,13 +209,15 @@ def configure() {
...
 		zigbee.enrollResponse()
 	if (isEcolink()) {
 		cmds += configureEcolink()
 	} else if (isBoschRadionMultiSensor()) {
 		cmds += zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER, IAS_ZONE_TYPE_ATTRIBUTE)
-	}
+    } else if (isFrientSensor()) {
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: Please use tabs as indentation

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy (hunk 1 above)
Comment (PKacprowiczS) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: Also please change deviceJoinName to generic name, like `frient Open/Closed Sensor`

Comment (rboy1) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: It looks like the Dawon fingerprint was recently added and hence missed from the baseline used. It has been fixed and added to the commit.

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy (hunk 2 above)
Comment (rboy1) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: Fixed

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy (hunk 3 above)
Comment (rboy1) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: Fixed

**Comment by frient-design** [comment]
We have made all the requested changes, can you please accept the request to continue the certification or let us know if more modification is needed

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 4]
```diff
@@ -237,4 +240,16 @@ private Boolean isEcolink() {
...
 	device.getDataValue("manufacturer") == "Bosch" && device.getDataValue("model") == "RFMS-ZBMS"
-}
\ No newline at end of file
+}
+
+private Boolean isFrientSensor() {
+	device.getDataValue("manufacturer") == "frient A/S"
+}
+
+private Map temperatureEndpoint() {
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:249]: Wrong indentation in this method.

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy (hunk 4 above)
Comment (rboy1) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:249]: Fixed

**Review by rboy1 — COMMENTED** [review]
Fixed

**Review by rboy1 — APPROVED** [review]

**Comment by frient-design** [comment]
All the comments are fixed, can you please approve the request or let us know if any issues

**Review by PKacprowiczS — APPROVED** [review]
@greens

**Review by KKlimczukS — APPROVED** [review]

**Pull request merged**